pkg/services/session/storage/temporary/storage.go

```diff
@@ -53,7 +53,7 @@ func (s *TokenStore) RemoveOld(epoch uint64) {
 	defer s.mtx.Unlock()
 
 	for k, tok := range s.tokens {
-		if tok.ExpiredAt() <= epoch {
+		if tok.ExpiredAt() < epoch {
```

Comment:
This differs with [docs](https://github.com/nspcc-dev/neofs-sdk-go/blob/71891029da36f61d1f64db47b7df84d8014adf1a/session/common.go#L222-L225).